Heat_method_3/test/Heat_method_3/Heat_method_traits_3.h

```diff
@@ -59,6 +59,11 @@ struct Heat_method_traits_3
     { return 0;}
   };
 
+  struct Compute_squared_length_3 {
```

Comment:
Can you point out where this functor is used.

Reply:
It is used to compute the `cotangent_3` that is used to compute the `cotangent_weight`.

Reply:
https://github.com/CGAL/cgal/blob/cac04ed1496a6c17c8c4b16dfa09c20999204595/Weights/include/CGAL/Weights/internal/utils.h#L273